theano/tensor/nnet/abstract_conv.py

```diff
@@ -782,7 +1151,7 @@ def conv3d_grad_wrt_weights(input,
                                                filter_flip=filter_flip,
                                                filter_dilation=filter_dilation)
 
-    return gradWeight_op(input, output_grad, filter_shape[:-3])
+    return gradWeight_op(input, output_grad, filter_shape[-3:])
```

Comment:
Yes, the [TestAssertShape](https://github.com/Theano/Theano/blob/7f1c36774f7c5163cf83790a44df6775244cf5da/theano/tensor/nnet/tests/test_abstract_conv.py#L319-L330) tests now call these `conv*_grad_wrt_*` functions. They don't do very extensive testing, but they do discover this typo.